buttercup.el

```diff
@@ -1536,46 +1548,55 @@ EVENT and ARG are described in `buttercup-reporter'."
     (pcase event
       (`buttercup-started
        (setq buttercup-reporter-batch--start-time (float-time)
-             buttercup-reporter-batch--failures nil)
+             buttercup-reporter-batch--failures nil
+             ;; Don't put any output on hold if nothing is ever omitted.
+             buttercup--pending-output (not buttercup-silent-skipping))
```

Comment:
buttercup--pending-output should be defined before this defun.

Reply:
Fixed.